src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilBarnPåBehandling/LeggTilBarnPåBehandling.tsx

```diff
@@ -119,29 +116,39 @@ const LeggTiLBarnPåBehandling: React.FC<IProps> = ({ behandling }) => {
             <Dropdown.Menu.List.Item onClick={() => settVisModal(true)}>
                 Legg til barn
             </Dropdown.Menu.List.Item>
-            <UIModalWrapper
-                modal={{
-                    tittel: (
-                        <LeggTilBarnLegend>
-                            Legg til barn
-                            <HelpText style={{ marginLeft: '0.5rem' }}>
+            {visModal && (
+                <Modal open onClose={onAvbryt} aria-label={'Legg til barn'} width={'35rem'} portal>
+                    <Modal.Header>
+                        <ModalHeaderFlex>
```

Comment:
Her lurer jeg på om vi kan bruke HStack med gap i stedet for en stylet div https://aksel.nav.no/komponenter/primitives/hstack

Reply:
Ooh, snasent! det må jeg teste ut

Reply:
Funket fjell: e1a013c3e71b33aa4d4127a493d2895fee7f0d90